packages/wonder-blocks-core/components/no-ssr.js

```diff
@@ -41,6 +26,23 @@ type State = {|
     mounted: boolean,
 |};
 
+/**
+ * Defer or change rendering until the component did mount.
```

Comment:
What's the rationale for moving doc strings to be after Props?

Reply:
This attaches the docs to the type itself, rather than the file. I can see that being useful.